errors.md

```diff
@@ -1,42 +1,47 @@
-# Error Handling
+# Penanganan Eror
 
-- [Introduction](#introduction)
-- [Configuration](#configuration)
-- [The Exception Handler](#the-exception-handler)
-    - [Reporting Exceptions](#reporting-exceptions)
-    - [Exception Log Levels](#exception-log-levels)
-    - [Ignoring Exceptions By Type](#ignoring-exceptions-by-type)
-    - [Rendering Exceptions](#rendering-exceptions)
-    - [Reportable & Renderable Exceptions](#renderable-exceptions)
-- [HTTP Exceptions](#http-exceptions)
-    - [Custom HTTP Error Pages](#custom-http-error-pages)
+- [Penanganan Eror](#penanganan-eror)
```

Comment:
konten pada daftar isi tidak sesuai dengan file aslinya. hal ini disebabkan oleh ekstensi All In Markdown milik Yu Zhen, saya rekomendasikan untuk dimatikan.

Reply:
telah diselesaikan pada: https://github.com/lara-army/dokumentasi-laravel/pull/104/commits/54a4120be1aa4d2adf647fabb5e2d1dedcf90457

Reply:
waduh makasih